flytekit/core/base_task.py

```diff
@@ -521,6 +524,23 @@ def dispatch_execute(
                         f"Failed to convert return value for var {k} for function {self.name} with error {type(e)}: {e}"
                     ) from e
 
+            INPUT = "input"
+            OUTPUT = "output"
+            from flytekit import Deck
```

Comment:
same here import from the original place where this is defined.  I don't think it needs to be here instead of the top of the file...